github-connector/chart/Deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: github-connector
+spec:
+  selector:
+    matchLabels:
+      app: github-connector
+  template:
+    metadata:
+      labels:
+        app: github-connector
+    spec:
+      containers:
+      - name: github-connector
+        image: karoljaksik/webhook-test:1.0.3
+        resources:
+          limits:
+            memory: "128Mi"
+            cpu: "500m"
+        ports:
+        - containerPort: 8080
+        env:
+        - name: PORT
+          value: ":8080"
```

Comment:
The port number here should be an integer value (simply `8080`), you can add the colon in the code.